src/app/app.component.ts

```diff
@@ -75,6 +76,10 @@ export class AppComponent implements AfterViewInit, OnDestroy, OnInit {
     private matomoTracker: MatomoTracker,
     private router: Router,
   ) {
+    this.isPlatformBrowser = isPlatformBrowser(this.platformId);
```

Comment:
My editor was complaining about the old position of this. In the constructor seems like a normal place to do it.

Reply:
Right, phpstorm doesn't complain but I think vscode rightly says "Property 'platformId' is used before its initialization.ts(2729)".